python-frontend/src/test/java/org/sonar/python/tree/IPythonTreeMakerTest.java

```diff
@@ -39,25 +38,36 @@ public class IPythonTreeMakerTest extends RuleTest {
   public void line_magic() {
     var parse = parseIPython("print(b)\n" +
       "a = %alias showPath pwd && ls -a\n", treeMaker::fileInput);
-    assertThat(parse).isNotNull();
-    var lineMagic = findFirstChildOf(parse, Tree.Kind.LINE_MAGIC);
+    var statementList = findFirstChildWithKind(parse, Tree.Kind.STATEMENT_LIST);
+    assertThat(statementList.children()).hasSize(2);
+    assertThat(statementList.children().get(0).getKind()).isEqualTo(Tree.Kind.EXPRESSION_STMT);
+    assertThat(statementList.children().get(1).getKind()).isEqualTo(Tree.Kind.ASSIGNMENT_STMT);
+    var lineMagic = findFirstChildWithKind(statementList.children().get(1), Tree.Kind.LINE_MAGIC);
```

Comment:
Probably here it's a bit better to directly use the API from `AssignmentStatement#assignedValue` to check that the RHS is a line magic, instead of the helper from the tests.